importers/tests/test_serializers.py

```diff
@@ -60,24 +63,22 @@ def test_fixtures_import(self):
         # this is kind of a crappy test, but i'm just using implicit knowledge of fixtures
         # to test that the entries created are correct
         results = self.sanitizer.get_sanitized_dataframe()
-
-        # TODO - refactor this so it sucks less, ideally create it and then review individual data points
         self.sanitizer.save_results(results)
 
         supplement_events_exist = SupplementEvent.objects.all().exists()
         self.assertTrue(supplement_events_exist)
 
     def test_get_measurement_unit_and_quantity_from_name(self):
         test_name_1 = 'Snake Oil (200mg)'
-        result = ExcelSupplementFileSerializer.get_measurement_and_quantity_from_name(test_name_1)
+        result = ExcelSupplementFileSerializer._get_measurement_and_quantity_from_name(test_name_1, 'random_uuid')
```

Comment:
You forced this just to make the test pass, but taking a step back, the original function was correct - _get_measurement_and_quantity_from_name should only need the name and not some janky uuid/string passed in it.